week3/57-skhiema-gorniera/task.py

```diff
@@ -0,0 +1,15 @@
+def main():
+    n, x = int(input()), float(input())
```

Comment:
make method to calc result. method gets n, x and InputStream as input. method returns number as a result